pkgs/applications/editors/jetbrains/patches/jetbrains-remote-dev.patch

```diff
@@ -1,17 +1,36 @@
 --- a/plugins/remote-dev-server/bin/launcher.sh
 +++ b/plugins/remote-dev-server/bin/launcher.sh
-@@ -366,6 +366,8 @@
+@@ -218,10 +218,11 @@
+   exit 1
+ fi
+
++
+ # ---------------------------------------------------------------------
+ # Configure fonts and fontconfig
+ # ---------------------------------------------------------------------
+-if [ $IS_DARWIN -ne 1 ]; then
++if false; then
+   FONTS_CONFIGURATION_BASE_PATH="$REMOTE_DEV_SERVER_DIR/selfcontained/fontconfig"
+   if [ ! -d "$FONTS_CONFIGURATION_BASE_PATH" ]; then
+     echo "ERROR! Unable to locate font configuration source directory in self-contained distribution: '$FONTS_CONFIGURATION_BASE_PATH'." 1>&2
+@@ -366,7 +370,9 @@
+ else
    REMOTE_DEV_SERVER_USE_SELF_CONTAINED_LIBS=1
  fi
 
+-if [ $REMOTE_DEV_SERVER_USE_SELF_CONTAINED_LIBS -eq 1 ]; then
 +REMOTE_DEV_SERVER_USE_SELF_CONTAINED_LIBS=0
 +
- if [ $REMOTE_DEV_SERVER_USE_SELF_CONTAINED_LIBS -eq 1 ]; then
++if false; then
    SELFCONTAINED_LIBS="$REMOTE_DEV_SERVER_DIR/selfcontained/lib"
```

Comment:
This change doesn't appear to be required, since the check would always be false anyway with the existing patch.

Reply:
In some IDEs (rust-rover, if I recall correctly) the variable `REMOTE_DEV_SERVER_USE_SELF_CONTAINED_LIBS` is named differently. So I avoided mentioning the variable name on line 24.

Reply:
Yeah, the fact that they all have different launcher scripts makes this harder than it has to be....

I wonder if you could replace the current patch with one that just disables the problematic fontconfig section, and then use a wrapper to set `REMOTE_DEV_SERVER_USE_SELF_CONTAINED_LIBS=0` by default.

Reply:
> Yeah, the fact that they all have different launcher scripts makes this harder than it has to be....
> 
> I wonder if you could replace the current patch with one that just disables the problematic fontconfig section, and then use a wrapper to set `REMOTE_DEV_SERVER_USE_SELF_CONTAINED_LIBS=0` by default.

This variable (`REMOTE_DEV_SERVER_USE_SELF_CONTAINED_LIBS`) is assigned within the script, so I guess a wrapper would not help here.

Reply:
I'm pretty sure that it only does so if it isn't already set, the variable is documented as part of the public interface:

>   echo "  REMOTE_DEV_SERVER_USE_SELF_CONTAINED_LIBS  set to '0' to skip using bundled X11 and other linux libraries from plugins/remote-dev-server/selfcontained. Use everything from the system."
>   echo "                                             by default bundled libraries are used"

Reply:
I just talked to @NickCao and he preferred moving env var to wrapper in another separate pr.

Reply:
Yeah since everyone seems to have different opinions on the ` REMOTE_DEV_SERVER_USE_SELF_CONTAINED_LIBS` situation, let's bring the fontconfig fix in first.